polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -911,23 +912,16 @@ struct GPUReturnOpLowering : public ConvertOpToLLVMPattern<gpu::ReturnOp> {
 
 struct ConvertPolygeistToLLVMPass
     : public impl::ConvertPolygeistToLLVMBase<ConvertPolygeistToLLVMPass> {
-  ConvertPolygeistToLLVMPass() = default;
-  ConvertPolygeistToLLVMPass(bool useBarePtrCallConv, bool emitCWrappers,
-                             unsigned indexBitwidth, bool useAlignedAlloc,
-                             const llvm::DataLayout &dataLayout) {
-    this->useBarePtrCallConv = useBarePtrCallConv;
-    this->emitCWrappers = emitCWrappers;
-    this->indexBitwidth = indexBitwidth;
-    this->dataLayout = dataLayout.getStringRepresentation();
-  }
+  using impl::ConvertPolygeistToLLVMBase<
+      ConvertPolygeistToLLVMPass>::ConvertPolygeistToLLVMBase;
 
   void runOnOperation() override {
     ModuleOp m = getOperation();
     const auto &dataLayoutAnalysis = getAnalysis<DataLayoutAnalysis>();
 
     LowerToLLVMOptions options(&getContext(),
                                dataLayoutAnalysis.getAtOrAbove(m));
-    options.useBarePtrCallConv = useBarePtrCallConv;
+    options.useBarePtrCallConv = true;
```

Comment:
Always `true`

Reply:
We're asserting the type converter has that option set. This is not the option from our pass, but from the generic LLVM conversion options.